src/main/cpp/blaze_util_linux.cc

```diff
@@ -21,6 +21,9 @@
 #include <sys/statfs.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <stdio.h>
```

Comment:
Thanks for trying to match the style, but just use C++ headers.  Remove the redundant errno.h.